### DISCUSSION TITLE
Adding current date to custom instructions

### DISCUSSION THREAD

**Discussion by adieyal** [agno-agi/agno#51] [Ideas]
It could be useful to add a flag to the assistant's custom instructions to include the current date to enable queries relative to time, e.g. _Where there any notable trends in last month's sales?_

I've added mine to extra_instructions, i.e.
```python
extra_instructions=[f"The current date is {datetime.now()}"]
```

but it might be a common enough requirement to warrant adding as a bool on Assistant
```python
class Assistant(BaseModel):
    ...
    add_current_time: bool = False
```

**Comment by ashpreetbedi** [comment]
This is a great idea @adieyal, @ysolanky  do you want to implement this?

**Reply by ysolanky** [reply]
@ashpreetbedi Yes. This sounds like a great feature to add.

**Comment by adieyal** [comment]
Sure here it is - https://github.com/phidatahq/phidata/pull/55
